docs/2_4_common_schema.adoc

```diff
@@ -725,7 +727,7 @@ latexmath:[\mathbf{T}_{interval} = ] `intervalCounter / resolution`.
 
 This value must be greater than 0.
 
-This attribute is allowed if <<supportsFraction,`supportsFraction = true`>> and is required if <<intervalVariability,intervalVariability == `constant`>> or <<intervalVariability,intervalVariability == `fixed`>>.
+This attribute is allowed if <<supportsFraction,`supportsFraction = true`>> and is required if <<intervalVariability,intervalVariability == `constant`>> and attribute <<intervalDecimal>> is not used.
```

Comment:
@andreas-junghanns I kept this and do not mention the fixed, calculated and tunable case, because even if a start value for intervalCounter is allowed, it is _not required_ to have one, right?
I added an analogous remark (required if intervalCounter not used) to the intervalDecimal attribute to have it consistently.